### PR TITLE
CortexXDRIR updated the mirror in time range

### PR DESCRIPTION
**Pull request by MosheEichler**
## Related Issues
fixes: [XSUP-41478](https://jira-dc.paloaltonetworks.com/browse/XSUP-41478)
fixes: [XSUP-43604](https://jira-dc.paloaltonetworks.com/browse/XSUP-43604)
fixes: [XSUP-41382](https://jira-dc.paloaltonetworks.com/browse/XSUP-41382)

## Description
Update the the mirror in time range to include the all time until the mirror triggered minus the XDR delay (which will be mirrored in the next iteration).

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/6_1_88.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Fixed an issue where closing an incident on XDR doesnt closed on XSOAR.
```
Comment (dorschw) [inline Packs/CortexXDR/ReleaseNotes/6_1_88.md:6]:
```suggestion
- Fixed an issue where mirroring-in would not apply to all incidents.
- Fixed an issue where closing an incident on XDR wouldn't close the matching XSOAR incident.
```

File: Packs/CortexXDR/ReleaseNotes/6_1_88.md [hunk 2]
```diff
@@ -3,6 +3,6 @@
 
 ##### Palo Alto Networks Cortex XDR - Investigation and Response
 
+
 - Fixed an issue with outgoing mirroring when closing an incident in Cortex XSOAR with *close_xdr_incident* set to False resulted in a 400 error.
 - Updated the Docker image to: *demisto/python3:3.11.10.115887*.
```
Comment (shmuel44) [inline Packs/CortexXDR/ReleaseNotes/6_1_88.md:8]:
```suggestion
- Fixed an issue with outgoing mirroring when closing an incident in Cortex XSOAR with *close_xdr_incident* set to False resulted in a 400 error.
- Updated the Docker image to: *demisto/python3:3.11.10.115887*.

```

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 3]
```diff
@@ -908,9 +907,8 @@ def get_modified_remote_data_command(client, args, mirroring_last_update: str =
         lte_modification_time_milliseconds=lte_modification_time_milliseconds,
         limit=100)
     last_run_mirroring = (lte_modification_time_milliseconds + timedelta(milliseconds=1))
-    # Format with milliseconds as string, truncate microseconds
-    last_run_mirroring_str = last_run_mirroring.replace(tzinfo=pytz.UTC).strftime(  # type: ignore
-        '%Y-%m-%d %H:%M:%S.%f')[:-3] + '+02:00'  # type: ignore
+
+    last_run_mirroring_str = last_run_mirroring.strftime('%Y-%m-%d %H:%M:%S.%f')
```
Comment (dorschw) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:911]: not truncating microseconds anymore, on purpose?

File: Packs/CortexXDR/ReleaseNotes/6_1_93.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Fixed an issue where mirroring-in would not apply to all incidents.
+- Fixed an issue where closing an incident on XDR wouldn't close the matching XSOAR incident.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/ReleaseNotes/6_1_93.md:7]:
```suggestion
- Fixed an issue where closing an incident in Cortex XDR wouldn't close the matching Cortex XSOAR incident.
```

**Comment by ShirleyDenkberg** [comment]
@shmuel44 @maimorag @dorschw Doc review completed.

**Review by dorschw — APPROVED** [review]
Great job. Please add given/when/then to the unit tests and let's ship this

**Pull request merged**